.github/workflows/ci.yml

```diff
@@ -118,3 +118,39 @@ jobs:
           dotnet help
             test "$(ls -A /var/log/opentelemetry/dotnet )"
         '
+
+  pack:
+    needs: build
+
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3.1.0
+      - uses: nuget/setup-nuget@v1.0.7
+
+      - name: Download Windows Artifacts from build job
+        uses: actions/download-artifact@v3
```

Comment:
Can you please pin the minor and patch versions of the GH Actions to have (in theory) more stable builds?